PAC/common/imitation_TE.cpp

```diff
@@ -0,0 +1,54 @@
+#include "imitation_TE.h"
+
+float imitation_TE::get_random() {
+  max = get_max();
+  min = get_min();
+  std::vector<float> iv(11);
+  std::vector<float> fv(10);
+  std::iota(iv.begin(), iv.end(), min);
+  std::iota(fv.begin(), fv.end(), 1);
+  unsigned int index_fv = get_index();
+  unsigned int index_iv = get_index();
+  return fv.at(index_fv) * 0.1f + iv.at(index_iv);
+}
+
+void imitation_TE::set_max(float max_in_range) { this->max = max_in_range; }
```

Comment:
Для более чёткой прослеживаемости логического блока кода необходимо перенести на новую строку открывающие и закрывающие фигурные скобки.

Например:

```suggestion
void imitation_TE::set_max(float max_in_range) 
   { 
      this->max = max_in_range; 
   }
```